modules/ebi-metagenomics/eggnogmapper/meta.yml

```diff
@@ -48,16 +53,23 @@ output:
       description: |
         Groovy Map containing sample information
         e.g. `[ id:'test', single_end:false ]`
-
-  - csv:
+  - hits:
       type: file
-      description: CSV containing protein annotations
-      pattern: "*.emapper.hits"
-
+      description: Tabulated file containing hits
+      pattern: "*.hits"
+  - annotations:
+      type: file
+      description: Tabulated file containing annotations
+      pattern: "*.annotations"
+  - orthologs:
+      type: file
+      description: Tabulated file containing EggNOG orthologs
+      pattern: "*.emapper.orthologs"
```

Comment:
...and *orthologs here? Just make sure you follow the same format as in the main.nf outputs